package.json

```diff
@@ -19,9 +19,9 @@
   },
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
-    "arsenal": "github:scality/Arsenal#2b9ac57",
+    "arsenal": "github:scality/Arsenal#c5e21e0",
     "async": "~2.5.0",
-    "aws-sdk": "2.178.0",
+    "aws-sdk": "^2.178.0",
```

Comment:
Noticed the `^` - I believe it accidentally came from my PR, and we should be good as we're gonna merge the aws-sdk version PR first.